homeassistant/components/vicare/manifest.json

```diff
@@ -11,5 +11,5 @@
   "documentation": "https://www.home-assistant.io/integrations/vicare",
   "iot_class": "cloud_polling",
   "loggers": ["PyViCare"],
-  "requirements": ["PyViCare==2.21.0"]
+  "requirements": ["PyViCare==2.25.0"]
```

Comment:
This should be a standalone preliminary PR

Reply:
here it is: https://github.com/home-assistant/core/pull/90536